lib/govuk_app_config/govuk_prometheus_exporter.rb

```diff
@@ -1,14 +1,34 @@
 module GovukPrometheusExporter
   def self.configure
-    unless Rails.env == "test" || (ENV["GOVUK_PROMETHEUS_EXPORTER"]) != "true"
+    if (ENV["GOVUK_PROMETHEUS_EXPORTER"]) == "true"
+      return if defined?(Rails) ? Rails.env == "test" : false
```

Comment:
Highly recommend factoring the condition out into a separate function to make the logic clearer. Something like

```ruby
def self.should_configure
  ENV['GOVUK_PROMETHEUS_EXPORTER'] == 'true' && defined? Rails && Rails.env != 'test'
end

def self.configure
  return unless should_configure

  require 'prometheus_exporter'
...
```